components/rhel/rhel-ose/Vagrantfile

```diff
@@ -87,8 +87,8 @@ Vagrant.configure(2) do |config|
     echo "You can now access the OpenShift console on: https://${OSIP}:8443/console"
     echo
     echo "To use OpenShift CLI, run:"
-    echo "$ vagrant ssh"
-    echo "$ oc login"
+    echo "vagrant service-manager install-cli openshift"
+    echo "$ oc login ${OSIP}:8443"
```

Comment:
I think we should rename `OSIP` to `OS_IP` which reads better and at least give user hint that is it IP of OS (openshift)